apps/climatemappedafrica/src/components/ExplorePage/useLayerStyles.js

```diff
@@ -0,0 +1,74 @@
+import {
+  CHART_PRIMARY_COLOR_SCHEME,
+  CHART_SECONDARY_COLOR_SCHEME,
+} from "@/climatemappedafrica/theme";
```

Comment:
`apps` can and should use `packages` but `packages` **must never** use `apps`... they can use other packages e.g. this package can use `@commons-ui/core` for example.